tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -46,6 +47,76 @@ type Files = { [key: string]: string | Buffer };
 type Params = { [key: string]: string | number | boolean | string[] };
 type Env = { [key: string]: string | number | boolean | undefined };
 
+type ChildParams = {
+  command: string[],
+  cwd?: string,
+  env?: Env,
+  shell?: boolean,
+  sendSIGINTAfter?: number,
+};
+
+class Child {
+  params: ChildParams;
+  process: ChildProcess;
+  output = '';
+  exited: Promise<number>;
+
+  constructor(params: ChildParams) {
+    this.params = params;
+    this.process = spawn(params.command[0], params.command.slice(1), {
+      env: {
+        ...process.env,
+        ...params.env,
+      } as any,
+      cwd: params.cwd,
+      shell: params.shell,
+    });
+    if (process.env.PW_RUNNER_DEBUG)
+      process.stdout.write(`\n\nLaunching ${params.command.join(' ')}\n`);
```

Comment:
Shouldn't it write to stderr?

Reply:
Well, it's not an error but a debug log, so I'd say stdout is right.